flink-table/flink-table-common/src/main/java/org/apache/flink/table/utils/TypeMappingUtils.java

```diff
@@ -169,8 +169,7 @@ public static void checkPhysicalLogicalTypeCompatible(
 										logicalFieldName,
 										physicalFieldType,
 										physicalFieldName,
-										"TableSource return type"),
-								cause));
+										"TableSource return type"), cause));
```

Comment:
What I meant is that we can inline the `TableSource return type` into the string. There is no reason for having it as an argument to `String.format()`.:
```
(cause) -> new ValidationException(
						String.format("Type %s of table field '%s' does not match with " +
							"the physical type %s of the '%s' field of the "TableSource return type".",
								logicalFieldType,
								logicalFieldName,
								physicalFieldType,
								physicalFieldName),
						cause));
```

Reply:
Oh, sry, you are right.